### PR TITLE
EVA-185: Species Specific Notifications on Variant Browser for what version VEP and what Ensembl cache used

### PR DESCRIPTION
**Pull request by kandaj**
We need to install species specific notifications on Variant Browser for what version VEP and what Ensembl cache were used.
I have updated the folder structure at:
/nfs/production2/eva/VEP/VEP_Output/<species>/
Each VEP output file is now in a folder that is named <VEP v>_<Ensembl cache version>. We shall use only one VEP output file for each species so we should be able to use the file path to automatically generate a sentence for the website to explain which VEP version and cache was used for annotation.

### CODE REVIEW

File: src/js/clinvar/eva-clinvar-annotation-panel.js [hunk 1]
```diff
@@ -259,5 +253,24 @@ ClinvarAnnotationPanel.prototype = {
         }
 
         return annotPanel;
+    },
+    getVepNotificationText: function(species, annotationVerisons){
```
Comment (jmmut) [inline src/js/clinvar/eva-clinvar-annotation-panel.js]: typo verisons -> versions

File: tests/unit/variant_browser.js [hunk 2]
```diff
@@ -209,6 +209,32 @@ describe('Consequence Type Tree', function(){
...
+        },
+        {
+            "vepVersion": "89",
+            "cacheVersion": "35",
+            "defaultVersion": true
+        },
+        {
+            "vepVersion": "88",
+            "cacheVersion": "35",
+            "defaultVersion": true
```
Comment (jmmut) [inline tests/unit/variant_browser.js]: should this be false?

File: src/js/eva-annotation-model.js [hunk 3]
```diff
@@ -273,27 +273,27 @@ var consequenceTypeDetails = {
...
-    {species: 'agambiae_agamp4', text: 'Variant Effect Predictor (VEP) v78 annotation against the Ensembl Genomes v26 geneset.', vep_version:78},
-    {species: 'aminimus_1v1', text: 'Variant Effect Predictor (VEP) v87 annotation against the VectorBase v1606 geneset.', vep_version:87},
-    {species: 'aquadriannulatus_quad4av1', text: 'Variant Effect Predictor (VEP) v82 annotation against the VectorBase v1606 geneset.', vep_version:82},
-    {species: 'asinensis_v1', text: 'Variant Effect Predictor (VEP) v82 annotation against the VectorBase v1606 geneset.', vep_version:82},
-    {species: 'astephensi_sda500v1', text: 'Variant Effect Predictor (VEP) v82 annotation against the VectorBase v1606 geneset.', vep_version:82},
-    {species: 'athaliana_tair10', text: 'Variant Effect Predictor (VEP) v87 annotation against the Ensembl Genomes v34 geneset.', vep_version:87},
-    {species: 'btaurus_umd31', text: 'Variant Effect Predictor (VEP) v82 annotation against the Ensembl v82 geneset.', vep_version:82},
+    {species: 'aaegypti_aaegl3', text: 'Variant Effect Predictor (VEP) v{vepVersion} annotation against the Ensembl Genomes v{cacheVersion} geneset.', vep_version:82},
```
Comment (jmmut) [inline src/js/eva-annotation-model.js]: agreed offline to remove the vep_version attribute here

**Comment by kandaj** [comment]
changes made.

File: tests/acceptance/variant_browser.js [hunk 4]
```diff
@@ -460,6 +466,19 @@ function variantFilterByMAF(driver){
     return driver;
 }
 
+function checkAnnotationNotiifcation(driver){
```
Comment (cyenyxe) [inline tests/acceptance/variant_browser.js]: Typo in `notification`

File: tests/unit/variant_browser.js [hunk 5]
```diff
@@ -209,6 +209,22 @@ describe('Consequence Type Tree', function(){
...
+        "defaultVersion": true
+    };
 
+    it('should return empty for chircus_10', function(){
+        expect(annotationPanel.getVepNotificationText('chircus_10',testObjectBefore)).to.deep.equal('');
+    });
+    
+    it('should return  "Variant Effect Predictor (VEP) v89 annotation against the Ensembl Genomes v35 geneset" for aaegypti_aaegl3', function(){
+        var expectedObjectAfter = "Variant Effect Predictor (VEP) v89 annotation against the Ensembl Genomes v35 geneset.";
+        expect(annotationPanel.getVepNotificationText('aaegypti_aaegl3',testObjectBefore)).to.deep.equal(expectedObjectAfter);
```
Comment (cyenyxe) [inline tests/unit/variant_browser.js]: This test will start failing as soon as the web service returns a newer version of the annotation. The web service response should be mocked if possible.

Comment (kandaj) [inline tests/unit/variant_browser.js]: the function getVepNotificationText() accepts one annotation version. it replaces the version numbers in text.

**Pull request merged**